db/migrate/20161209162947_add_virt_who_to_katello_pools.rb

```diff
@@ -0,0 +1,9 @@
+class AddVirtWhoToKatelloPools < ActiveRecord::Migration
+  def up
+    add_column :katello_pools, :virt_who, :boolean, :default => false, :null => false
```

Comment:
Fair question, I'm always hesitant to add them out of the gate to reduce index explosion. However, we tend to be a read heavy and not a write heavy application so I can add one if we think this would gain us performance.